csaf_2.0/prose/csaf-v2-editor-draft.md

```diff
@@ -5696,15 +5696,34 @@ Secondly, the program fulfills the following for all items of:
           CVSS:3.1/AV:N/AC:L/PR:N/UI:N/S:U/C:N/I:N/A:H => 3.1
         ```
 
-    2. Retrieve the CVSS version from the CVSS namespace, if present. The CVRF CSAF converter outputs a warning that this value was guessed from the namespace. If more than one CVSS namespace is present and the element is not clearly defined via the namespace, this step MUST be skipped without a decision.
+    2. Retrieve the CVSS version from the CVSS element's namespace, if present. The CVRF CSAF converter outputs a warning that this value was guessed from the element's namespace.
+
+        *Example XYZ:*
+
+        ```
+          xmlns:cvssv31="https://www.first.org/cvss/cvss-v3.1.xsd"
+          <!-- -->
+          <cvssv31:ScoreSetV3>
+        ```
+
+        is handled the same as 
+
+        *Example XYZ:*
+
+        ```
+          <ScoreSetV3 xmlns="https://www.first.org/cvss/cvss-v3.1.xsd">
+        ```
+
+    
+    3. Retrieve the CVSS version from the CVSS namespace given in the root element, if present. The CVRF CSAF converter outputs a warning that this value was guessed from the global namespace. If more than one CVSS namespace is present and the element is not clearly defined via the namespace, this step MUST be skipped without a decision.
 
         *Example XYZ:*
 
         ```
           xmlns:cvssv3="https://www.first.org/cvss/cvss-v3.0.xsd" => 3.0
         ```
 
-    3. Retrieve the CVSS version from a config value, which defaults to `3.0`. (As the spec only mentions 3.0.) The CVRF CSAF converter outputs a warning that this value was taken from the config.
+    4. Retrieve the CVSS version from a config value, which defaults to `3.0`. (As the spec only mentions 3.0.) The CVRF CSAF converter outputs a warning that this value was taken from the config.
```

Comment:
Maybe replace `As the spec only mentions 3.0. ` with `As CSAF CVRF v1.2 predates CVSS v3.1` Or similar. The itch I try to scratch is the ambiguity of the term `the spec` Inside another specification (Russian doll)

Reply:
Done in [44eab57](https://github.com/oasis-tcs/csaf/pull/424/commits/44eab5729e15727cd089a785f8c01427920ba02e)